docs/pages/eas/json.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: eas.json
+description: Learn about available properties for EAS Build and EAS Submit to configure and override their default behavior from within your project.
+maxHeadingDepth: 4
+---
+
+import { EasJsonPropertiesTable } from '~/components/plugins/EasJsonPropertiesTable';
+import { Collapsible } from '~/ui/components/Collapsible';
+
+import commonSchema from '~/public/static/schemas/unversioned/eas-json-build-common-schema.js';
+import androidSchema from '~/public/static/schemas/unversioned/eas-json-build-android-schema.js';
+import iosSchema from '~/public/static/schemas/unversioned/eas-json-build-ios-schema.js';
+import submitAndroidSchema from '~/public/static/schemas/unversioned/eas-json-submit-android-schema.js';
+import submitIosSchema from '~/public/static/schemas/unversioned/eas-json-submit-ios-schema.js';
+
+**eas.json** is the configuration file for EAS CLI and services. It is located at the root of a project next to the **package.json**. You can find the complete reference of all available schema properties for [EAS Build](/build/introduction) (`build`) and [EAS Submit](/submit/introduction) (`submit`) on this page.
+
+## EAS Build
+
+### Common properties for native platforms
+
+<EasJsonPropertiesTable schema={commonSchema} />
+
+### Android-specific options
+
+<EasJsonPropertiesTable schema={androidSchema} />
+
+### iOS-specific options
+
+<EasJsonPropertiesTable schema={iosSchema} />
+
+## EAS Submit
```

Comment:
same comment as the above re: brief description and collapsible example

Reply:
Done:

<img width="879" alt="CleanShot 2023-09-24 at 16 54 45@2x" src="https://github.com/expo/expo/assets/10234615/df11ca91-f04c-4574-905a-386c19375458">